imports/plugins/included/payments-paystack/server/i18n/ar.json

```diff
@@ -0,0 +1,27 @@
+[{
```

Comment:
DId you add this manually? If so, why?

Reply:
No I didn't,  just edited the file to accommodate new changes and updates needed to make the feature explicit.